lua/cfc_loadouts/client/cl_cfc_loadout.lua

```diff
@@ -162,17 +163,21 @@ function CFCLoadouts.openLoadout()
         net.Start( "CFC_Loadout_WeaponTable" )
         net.WriteTable( selectedWeapons )
         net.SendToServer()
+
+        EmitSound( Sound( "Weapon_357.Reload" ), LocalPlayer():GetPos(), 1, CHAN_AUTO, 1, 75, 0, 100 )
     end
 
     local resetSelectButton = vgui.Create( "DButton", panel1 )
     resetSelectButton:SetPos( 9.6 * uiScale, 510.3 * uiScale )
     resetSelectButton:SetSize( 144 * uiScale, 27 * uiScale )
-    resetSelectButton:SetText( "Select default loadout" )
+    resetSelectButton:SetText( "Reset to default" )
     CFCLoadouts.paintButton( resetSelectButton )
 
     resetSelectButton.DoClick = function()
         net.Start( "CFC_Loadout_Resetweapons" )
         net.SendToServer()
+
+        EmitSound( Sound( "Weapon_357.OpenLoader" ), LocalPlayer():GetPos(), 1, CHAN_AUTO, 1, 75, 0, 100 )
```

Comment:
Might be better to use [surface.PlaySound](https://wiki.facepunch.com/gmod/surface.PlaySound)

Reply:
Sure, I'll get to it